micronaut-langchain4j-core/src/main/java/io/micronaut/langchain4j/store/memory/chat/cassandra/CassandraChatMemoryStoreConfigurationProperties.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2017-2025 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.langchain4j.store.memory.chat.cassandra;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.annotation.Internal;
+
+import java.util.List;
+
+/**
+ * {@link ConfigurationProperties} implementation for {@link CassandraChatMemoryStoreConfiguration}.
+ */
+@Internal
+@ConfigurationProperties(CassandraChatMemoryStoreConfigurationProperties.PREFIX)
+public class CassandraChatMemoryStoreConfigurationProperties implements CassandraChatMemoryStoreConfiguration {
+    /**
+     * Default port.
+     */
+    public static final Integer DEFAULT_PORT = 9042;
+    /**
+     * Default table name.
+     */
+    public static final String DEFAULT_TABLE_NAME = "message_store";
+    public static final String PREFIX = "langchain4j.store-memory-chat.cassandra";
+    public static final boolean DEFAULT_ENABLED = true;
+    public static final String PROPERTY_ENABLED = PREFIX + ".enabled";
```

Comment:
these static final fields can be constants of the extended interface.

Reply:
[I moved the constants to the interfaces](https://github.com/micronaut-projects/micronaut-langchain4j/pull/173/commits/6a98d4a7b8b8602e7bf5bde279207429e9b673e9)